### DISCUSSION TITLE
Can I change C4 Relationship() Color

### DISCUSSION THREAD

**Discussion by dantasticdotcom** [mingrammer/diagrams#939] [Q&A]
Is it possible to change the color of the Relationship when using C4 nodes?  I'd like some to be red to denote if an integration between two Systems() are insecure.

**Comment by benbjamin24** [comment]
Relationship("some_label.", color="red")  works for me, or send as kwargs.

redge_attributes = {
        "style": "dashed",
        "color": "red"
	}
Relationship("some_label.", **redge_attributes)

**Reply by dantasticdotcom** [reply]
Thank you, this worked perfectly.  Are all the possible attributes listed at https://www.graphviz.org/doc/info/attrs.html?  I ask because the documentation has no reference to Relationships, just nodes, edges, graphs, and clusters.  That's where I got hung up; I didn't know you could use these attributes within Relationship().
